File: 0009-palindrome-number/0009-palindrome-number.java
class Solution {
    public boolean isPalindrome(int x) {
        String orig = Integer.toString(x);
        String rev = "";
        for (int i = Integer.toString(x).length() - 1; i >= 0; i--) {
            rev = rev + Integer.toString(x).substring(i, i+1);
        }
        return orig.equals(rev);
    }
}

File: 0009-palindrome-number/0009-palindrome-number.py
# class Solution:
#     def isPalindrome(self, x: int) -> bool:
#         if (x < 0):
#             return False
#         reversed = 0
#         x_original = x
#         while(x > 0):
#             reversed = (reversed * 10) + (x % 10)
#             x = x // 10
#         return reversed == x_original

class Solution:
    def isPalindrome(self, x: int) -> bool:
        return str(x)[::-1] == str(x)